flixel/util/FlxColor.hx

```diff
@@ -15,7 +15,7 @@ import flixel.system.macros.FlxMacroUtil;
  *
  * @author Joe Williamson (JoeCreates)
  */
-abstract FlxColor(Int) from Int from UInt to Int to UInt
+abstract FlxColor(Int) from Int from UInt to Int to UInt #if cpp from cpp.UInt8 to cpp.UInt8 #end
```

Comment:
Shouldn't this be removed? a color's components may be constructed from or deconstructed to UInt8s but a color itself should always a 32 bit number